src/controllers/hotkeys/ActionNames.hpp

```diff
@@ -355,6 +355,19 @@ inline const std::map<HotkeyCategory, ActionDefinitionMap> actionNames{
               .argumentsPromptHover = "Should the tabs be enabled, disabled, "
                                       "toggled, or live-only.",
           }},
+         {"toggleGlobalNotificationSuppression",
+          ActionDefinition{
+              .displayName = "Toggle muting of all notifications",
+              .argumentDescription = "[on, off, or toggle. default: toggle]",
+              .minCountArguments = 0,
+              .maxCountArguments = 1,
+              .possibleArguments{{"Toggle", {}},
+                                 {"Enable notification muting", {"on"}},
+                                 {"Disable notification muting", {"off"}}},
+              .argumentsPrompt = "New value:",
+              .argumentsPromptHover = "Should all highlight notifications be "
+                                      "enabled, disabled, or toggled.",
+          }},
```

Comment:
Bikeshed: Do we want this as a "negative" option? "Disable notification muting" is (kind-of) a double-negative - you're enabling notifications.

Reply:
Good point! Happy to run with "mute notifications" and "unmute notifications" 👍

Reply:
Adjusted in 513e190a